homeassistant/auth/auth_store.py

```diff
@@ -410,13 +478,27 @@ def _set_defaults(self) -> None:
         """Set default values for auth store."""
         self._users = OrderedDict()  # type: Dict[str, models.User]
 
-        # Add default group
-        all_access_group = models.Group(
-            name=INITIAL_GROUP_NAME,
-            policy=DEFAULT_POLICY,
-        )
-
         groups = OrderedDict()  # type: Dict[str, models.Group]
-        groups[all_access_group.id] = all_access_group
-
+        _add_system_admin_group(groups)
```

Comment:
It would be more readable if we return a new updated dict instead of mutating the existing dict in place.

Reply:
Creating new dictionaries while we know that this is never called otherwise (since it's an internal function) seems inefficient.  Changed it now to return the group and have it assign it where it's called. 
72a0180